### ISSUE TITLE
[HOLD][Cleanup] Remove workaround that filters out any actions keyed by sequenceNumber

### ISSUE THREAD

**Issue by roryabraham** [Expensify/App#14574]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

HOLD on https://github.com/Expensify/Expensify/issues/254606

This issue is a reminder to remove [this temporary hack](https://github.com/Expensify/App/blob/502fff22c76027bebfe9fada63a3b6e14b6eb36d/src/pages/home/report/ReportActionsView.js#L258-L270) from the code.

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by roryabraham** [comment]
This is marked as both monthly and daily but should be monthly

**Comment by roryabraham** [comment]
Actually, I change my mind and am making this a weekly again. Seeking some clarity in [this issue](https://github.com/Expensify/App/issues/14765#issuecomment-1419708358)

**Comment by MelvinBot** [comment]
@roryabraham Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by roryabraham** [comment]
Put up a PR that just adds a log line so that we can come back in a week or so and see if it's being hit or if we're safe to remove the hack.

**Comment by roryabraham** [comment]
Removing the reviewing label because I need to come back to this next week

**Comment by roryabraham** [comment]
Should not be overdue. I added a server log in the hack so we can check later this week if it's still being hit at all.

**Comment by roryabraham** [comment]
Still cleaning up some leftover places where reportActions are being sent keyed by sequenceNumber. Here are two cleanup PRs this will need to be on HOLD for:

- https://github.com/Expensify/Web-Expensify/pull/36484
- https://github.com/Expensify/Web-Expensify/pull/36554

**Comment by roryabraham** [comment]
Still seeing the log line being hit a lot:

https://www.expensify.com/_devportal/tools/logSearch/#sort=asc&size=10000&query=blob%3A%22Front-end%20filtered%20out%20reportAction%20keyed%20by%20sequenceNumber!%22%20AND%20timestamp%3A%5B2023-03-10T00%3A00%20TO%202023-03-11T23%3A59%5D

Need to investigate why

**Comment by roryabraham** [comment]
This happened to me as recently as _2023-03-08 19:16:07_, but the same report action which was triggering that log line appears to no longer be keyed by reportActionID.

**Comment by roryabraham** [comment]
I'm not sure how offline notifications could affect Onyx data, but [this block](https://github.com/Expensify/Web-Expensify/blob/9bef6c2f5f1f98b2a23138eb166cb64c4a17cfbb/lib/ReportUtils.php#L2182-L2184) looks potentially suspect?

**Comment by roryabraham** [comment]
Still getting hit a ton

**Comment by roryabraham** [comment]
I've scoured seemingly all the places in E/App and Web-Expensify that reference sequenceNumbers and couldn't figure out where this is happening. Asked for help in slack: https://expensify.slack.com/archives/C04DC6LU2UB/p1679341586293249

**Comment by MelvinBot** [comment]
This issue has not been updated in over ***15*** days. @roryabraham eroding to ***Monthly*** issue.

P.S. Is everyone reading this sure this is really a near-term priority? Be brave: if you disagree, go ahead and close it out. If someone disagrees, they'll reopen it, and if they don't: one less thing to do!

**Comment by roryabraham** [comment]
No update

**Comment by roryabraham** [comment]
No update

**Comment by roryabraham** [comment]
Putting this on HOLD for https://github.com/Expensify/Expensify/issues/254606

**Comment by roryabraham** [comment]
on HOLD

**Comment by roryabraham** [comment]
Still on HOLD

**Comment by roryabraham** [comment]
still on HOLD

**Comment by roryabraham** [comment]
Still on HOLD

**Comment by roryabraham** [comment]
On HOLD

**Comment by roryabraham** [comment]
on HOLD

**Comment by roryabraham** [comment]
on HOLD

**Comment by roryabraham** [comment]
on HOLD

**Comment by roryabraham** [comment]
no update

**Comment by roryabraham** [comment]
This is blocked on removing sequenceNumber from OldApp.

*suggestion:* as soon as C+ get access to OldApp, task one of them w/ removing sequenceNumber in favor of `lastRead`. No back-end changes should be needed for that part.
